programmers/신규-아이디-추천.js

```diff
@@ -0,0 +1,19 @@
+function solution(new_id) {
+  let answer = '';
+
+  answer = new_id
+    .toLowerCase()
+    .replace(/[^a-z0-9\-\_\.]/g, '')
+    .replace(/\.{2,}/g, '.')
+    .replace(/(^\.|\.$)/g, '');
```

Comment:
이렇게 변수명을 한 번에 선언해주시는게 더 좋을거같아요!
```suggestion
  let answer = new_id
    .toLowerCase()
    .replace(/[^a-z0-9\-\_\.]/g, '')
    .replace(/\.{2,}/g, '.')
    .replace(/(^\.|\.$)/g, '');
```